public/program.css

```diff
@@ -0,0 +1,53 @@
+.program-info {
+    font-size: large;
+    color: var(--main-blue-color);
+
+    padding: 0px 20px; 
+    margin: 0px 30px;
+}
+
+.program-container {
+    display: flex;
+    justify-content: center;
+    margin: 50px;
+}
+
+.program-item {
+    font-family: 'Noto Sans', sans-serif;
+    line-height: 1.6;
+
+    max-width: min(35vw, 400px);
+    border: 5px solid var(--main-blue-color);
+    padding: 10px 20px; 
+    margin: 30px;
+}
+
+.program-item img {
+    max-width: min(35vw, 400px);
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
FYI @mcipsen, this is the bit that modifies the styling on mobile. It just means whenever the screen is less than 600px wide, apply these special formatting rules.

Reply:
cool!